### PR TITLE
Fix step log bug

### PR DESCRIPTION
**Pull request by lhw362950217**

### CODE REVIEW

File: python/couler/couler/steps/sqlflow_step.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ def sqlflow(sql,
             exit_time_wait = env.get("SQLFLOW_WORKFLOW_EXIT_TIME_WAIT", "0")
 
         log_dir = path.dirname(log_file)
-        command = '''(mkdir -p %s && step -e "%s" && sleep %s) 2>&1 | tee %s''' % (  # noqa E501
-            log_dir, escape_sql(sql), exit_time_wait, log_file)
+        command = '''mkdir -p %s && (step -e "%s" 2>&1 | tee %s) && sleep %s''' % (  # noqa E501
```
Comment (Yancey0623) [inline python/couler/couler/steps/sqlflow_step.py]: Just a confusion, why this bug has not  block the e2e test on CI?

Comment (lhw362950217) [reply python/couler/couler/steps/sqlflow_step.py]: On CI, we did not set the SQLFLOW_WORKFLOW_STEP_LOG env var. So, it goes the first branch 
```
    if not log_file:
        command = '''step -e "%s"''' % escape_sql(sql)
    else:
        command = '''mkdir -p %s && (step -e "%s" 2>&1 | tee %s) && sleep %s''' % (  # noqa E501
```

Comment (Yancey0623) [reply python/couler/couler/steps/sqlflow_step.py]: Maybe we can enable the step logs on GitHub CI?

**Review by Yancey0623 — APPROVED** [review]
LGTM

**Pull request merged**